src/STW.ProcessingApi.Function/Triggers/ProcessingApiTrigger.cs

```diff
@@ -8,16 +12,27 @@ namespace STW.ProcessingApi.Function.Triggers;
 public class ProcessingApiTrigger
 {
     private readonly ILogger _logger;
+    private readonly IValidator _validator;
 
-    public ProcessingApiTrigger(ILogger<ProcessingApiTrigger> logger)
+    public ProcessingApiTrigger(ILogger<ProcessingApiTrigger> logger, IValidator validator)
     {
         _logger = logger;
+        _validator = validator;
     }
 
     [Function(nameof(ProcessingApiTrigger))]
     public HttpResponseData Run([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "test")] HttpRequestData request)
     {
         _logger.LogInformation($"{nameof(ProcessingApiTrigger)} function was invoked.");
+        using (var reader = new StreamReader(request.Body, Encoding.UTF8))
+        {
+            var requestBody = reader.ReadToEnd();
+            var validationRules = new List<IRule> { new ExampleRule() };
```

Comment:
The rules should maybe come from DI in the validation service rather than the main function

Reply:
moved to DI, in the ServiceCollectionExtensions class